### DISCUSSION TITLE
Exclude "Just Ride Outdoors?"

### DISCUSSION THREAD

**Discussion by Buccafresh** [philosowaffle/peloton-to-garmin#558] [Q&A]
Hi there, 

I was wondering if it is possible to exclude "Just Ride Outdoors" / Outdoor cycling when using this amazing tool to sync from Peloton to Garmin. Currently, I saw these were the available options below. However, I am noticing my sync is double-counting the "Just Ride Outdoors" that I do in the Peloton App on my Android phone. (I record my outside bike rides with my Garmin watch, but also the Peloton app on my phone to get the badges/peloton challenges, even though I prefer the interface on my Garmin watch much more. I start my phone, and toss it in my riding backpack.) Is it possible to add "OutdoorCycling" to the code?

**_My research from the published wiki_**

Exclude Workout Types
Example use cases:

You take a wide variety of Peloton classes, including meditation and you want to skip uploading meditation classes.
You want to avoid double-counting activities you already track directly on a Garmin device, such as outdoor running workouts.
The available values are:

  Cycling
  BikeBootcamp
  TreadmillRunning
  OutdoorRunning
  TreadmillWalking
  OutdoorWalking
  Cardio
  Circuit
  Strength
  Stretching
  Yoga
  Meditation

Please add **OutdoorCycling** If it's technically possible.

**Comment by philosowaffle** [comment]
Thanks for calling this out, I've created a feature request [over here](https://github.com/philosowaffle/peloton-to-garmin/issues/559) to track adding support for this.

**Reply by Buccafresh** [reply]
Thank you!
